package.json

```diff
@@ -33,7 +33,7 @@
         "tb": "npm run test:browser",
         "tf": "npm run test:fps",
         "tr": "npm run coverage && npm run report && open .coverage/index.html",
-        "storybook": "storybook -p 6006",
+        "storybook": "storybook dev -p 6006",
```

Comment:
Why was this changed? Just curious

Reply:
It is part of storybook 7 upgrade. The same is used on `npm start` so maybe this script is not needed